engine-precompiles/src/native.rs

```diff
@@ -580,9 +588,19 @@ impl<I: IO> Precompile for ExitToEthereum<I> {
 
 #[cfg(test)]
 mod tests {
+    use aurora_engine_types::{Cow, U256};
+    use evm::ExitError;
+
     use super::{exit_to_ethereum, exit_to_near};
     use crate::prelude::sdk::types::near_account_to_evm_address;
 
+    fn validate_amount(amount: U256) -> Result<(), ExitError> {
```

Comment:
But this function should be out of the test module. And it should replace checks on the 329 and 512 lines.